README.md

```diff
@@ -83,9 +88,10 @@ which will essentially:
 1. Create a working tree at `<root>`/`<location>`/`<name>`
 2. Fetch from `repo` and checkout the specified `rev`
 3. Symbolically link each `<location>`/`<name>` from `<root>`/`<link>` (if specified)
-4. Repeat for all nested working trees containing a config file
-5. Record the actual commit SHAs that were checked out (with `--lock` option)
-6. Run optional post-install scripts for each dependency
+4. Materialize only the selected paths from `sparse_paths`, if present in the config file
+5. Repeat for all nested working trees containing a config file
+6. Record the actual commit SHAs that were checked out (with `--lock` option)
+7. Run optional post-install scripts for each dependency
```

Comment:
This content is good, but I'm not sure if it's applicable to most users. This might make more sense in it's own `use-cases` page.

Reply:
done